presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/FunctionCallProvider.java

```diff
@@ -21,19 +21,19 @@
 import java.util.List;
 import java.util.Objects;
 
-import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.toSymbolReferences;
+import static com.google.common.collect.ImmutableList.toImmutableList;
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
-class FunctionCallProvider
+public class FunctionCallProvider
         implements ExpectedValueProvider<FunctionCall>
 {
     private boolean isWindowFunction;
     private final QualifiedName name;
     private final boolean distinct;
     private final List<PlanTestSymbol> args;
 
-    public FunctionCallProvider(boolean isWindowFunction, QualifiedName name, boolean distinct, List<PlanTestSymbol> args)
+    private FunctionCallProvider(boolean isWindowFunction, QualifiedName name, boolean distinct, List<PlanTestSymbol> args)
```

Comment:
Get rid of the public constructor below. It's not clear which it creates. The goal is to make this hard to use FCP incorrectly. Specifically, it should be hard to create an FCP without it being obvious which kind of function you're creating.